br/pkg/task/stream.go

```diff
@@ -52,6 +53,9 @@ import (
 )
 
 const (
+	flagYes                     = "yes"
+	flagDryRun                  = "dry-run"
+	flagUntil                   = "until"
```

Comment:
should ask PM confirm too...

Reply:
cc @IANTHEREAL 
(`--yes` and `--dry-run` may be hidden?)

Reply:
checked